extensions/wikia/EditPageLayout/js/editor/AceEditor.js

```diff
@@ -49,6 +49,7 @@ define(
 	function beforeEditorInit() {
 		$('.loading-indicator').remove();
 		$('#wpSave').removeAttr('disabled');
+		$('#InfoboxPreview').click(launchInfoboxPreview);
```

Comment:
@katanka So InfoboxPreview button is going to work only when AceEditor is being used (because handler is assigned in here - AceEditor.js). So my question is: is the button displayed only when the AceEditor is being used? Because from what I see conditions for displaying InfoboxPreview button are different than conditions for displaying actually AceEditor (isCodePage).